docs/gh-workflows.md

```diff
@@ -0,0 +1,63 @@
+# GitHub Workflows
+
+The CLP repo includes several GitHub workflows for automating container image builds, artifact
+builds, testing, and linting. We briefly describe each workflow below.
+
+## [clp-core-build](../.github/workflows/clp-core-build.yaml)
+
+This is the main workflow for building (Linux) container images containing CLP-core's dependencies
+in addition to building CLP-core and unit testing it. To minimize build times, the jobs in the
```

Comment:
First sentence is kinda clumsy:
"This is the main workflow for building (Linux) container images containing CLP-core's dependencies, 
in addition to building CLP-core and performing unit testing."